libraries/c_sdk/aws/defender/test/system/aws_iot_tests_defender_system.c

```diff
@@ -1,6 +1,6 @@
 /*
- * FreeRTOS Defender V3.0.1
- * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ * AWS IoT Defender V3.0.0
+ * Copyright (C) 2018 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
```

Comment:
The defender version in this repository is FreeRTOS Defender V3.0.1, so that should be the same for this file. Also, copyright date should be 2020

Reply:
See 93eb5a43d